src/Microsoft.Framework.PackageManager/Utils/NuGetPackageUtils.cs

```diff
@@ -75,6 +88,11 @@ internal static class NuGetPackageUtils
                     stream.Seek(0, SeekOrigin.Begin);
                     var nupkgSHA = Convert.ToBase64String(sha512.ComputeHash(stream));
                     File.WriteAllText(hashPath, nupkgSHA);
+
+                    if (performingParallelInstalls)
+                    {
+                        extractDirectory.MoveTo(targetPath);
```

Comment:
`extractDirectory` is the `DirectoryInfo` for `targetPath`.  as-is this shouldn't do anything.